src/app/pages/choose-account/components/accounts.tsx

```diff
@@ -21,25 +20,34 @@ import { usePressable } from '@app/components/item-hover';
 import { useNativeSegwitAccountIndexAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
 import { useStacksAccounts } from '@app/store/accounts/blockchain/stacks/stacks-account.hooks';
 import { StacksAccount } from '@app/store/accounts/blockchain/stacks/stacks-account.models';
-import { Title } from '@app/ui/components/typography/title';
 
-const loadingProps = { color: '#A1A7B3' };
-const getLoadingProps = (loading: boolean) => (loading ? loadingProps : {});
+// #4476 TODO - test / refactor this - I'm not sure what it does
+// seems like it makes the account title a different colour when loading
+// I re-wrote this as it was way over complicated but not sure if its needed
+// - changes color based on whether its loading or not
+// const loadingProps = { color: '#A1A7B3' };
+// const getLoadingProps = (loading: boolean) => (loading ? loadingProps : {});
```

Comment:
Not sure here either, it must be really old code. It is likely outdated. GitLens says the code was added 4 years ago, haha. It does look like it just change the color during loading.

Reply:
I tested it, changing the colour and I can't see any difference when loading or not. 

I'm going to delete it, on my head be it if it's an important thing, I'll just re-add it if we need to